addons/mail/static/src/components/chatter_topbar/chatter_topbar_tests.js

```diff
@@ -409,9 +410,8 @@ QUnit.test('composer state conserved when clicking on another topbar button', as
         "send message button should not be active"
     );
 
-    await afterNextRender(() => {
-        document.querySelector(`.o_ChatterTopbar_buttonAttachments`).click();
-    });
+    document.querySelector(`.o_ChatterTopbar_buttonAttachments`).click();
+    await nextAnimationFrame();
```

Comment:
Why the `afterNextRender()` have been replaced with `nextAnimationFrame`? The former was more robust and explicitly waited for re-render, while the latter is very fragile/leaks OWL rendering implementation detail/prone to runbot race conditions.

Reply:
Because it is a test checking that nothing changed, so there is no render (`useStore` was observing too much before).

Reply:
Ok